Chart.yaml

```diff
@@ -14,7 +14,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.0
+version: 0.2.1
```

Comment:
this is a breaking change to existing functionality and defaults, so it's more than a patch version bump.